erpnext/hr/doctype/expense_claim/expense_claim.json

```diff
@@ -365,13 +366,19 @@
    "label": "Total Taxes and Charges",
    "options": "Company:company:default_currency",
    "read_only": 1
+  },
+  {
+   "fieldname": "delivery_trip",
```

Comment:
Only show this field if it is set, keep it hidden if not set.

Reply:
Done